### PR TITLE
minikube-docker

### PR DESCRIPTION
**Pull request by HoChristinaWuiYan**
## What does this PR do?

- [x] Adds new functionality
- [ ] Alters existing functionality
- [ ] Fixes a bug
- [ ] Improves Documentation

A brief description of changes to implementation or controller behavior:

Motivation for change:

### Testing

- [ ] I used existing unit tests
- [x] I manually tested locally
- [ ] I will manually test in a canary deployment

Please list your manual testing steps (sample `yaml` file, commands, important checks):

### Checklist

- [x] The documentation is up to date
- [x] My code is sufficiently commented
- [x] I have tested to the best of my ability
- [x] I have signed my commit (see [Contributing Docs](../CONTRIBUTING.md))

### CODE REVIEW

**Comment by nikos912000** [comment]
@Devatoria @ptnapoleon we thought it'd be useful to add that command for those using docker instead of containerd.

Before [this PR](https://github.com/DataDog/chaos-controller/pull/401/files) we also had to change the build commands to not use `ctr` but it should be fine now.

File: Makefile [hunk 1]
```diff
@@ -117,6 +117,17 @@ minikube-start:
 		--extra-config=apiserver.enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota \
 		--iso-url=https://public-chaos-controller.s3.amazonaws.com/minikube/minikube-2021-01-18.iso
 
+minikube-start-docker:
```
Comment (ptnapoleon) [inline Makefile:120]: It seems the only difference is in `container-runtime`? If that's the case, could you model this command on `minikube-start-big`, to reduce duplication? thanks!

Comment (HoChristinaWuiYan) [reply Makefile]: Done!

**Review by ptnapoleon — APPROVED** [review]

**Comment by ptnapoleon** [comment]
Thanks! Very useful PR!

**Pull request merged**